noir-projects/noir-contracts/contracts/nft_contract/src/hacky_partial_note.nr

```diff
@@ -0,0 +1,19 @@
+use dep::aztec::{prelude::{NullifiableNote, PrivateContext}, macros::notes::note};
```

Comment:
This is the workaround used to get encrypted log of partial note.

Reply:
Why does it need to hav ethe same name? Because of the note type id? If so let's leave a comment.